boto3/examples/s3.rst

```diff
@@ -144,3 +144,20 @@ Boto3 will automatically compute this value for us.
                              SSECustomerAlgorithm='AES256')
     print("Done, response body:")
     print(response['Body'].read())
+
+
+Downloading a specific version of an S3 object
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
Let's make sure this is the same length as the header string above.

Reply:
Ah thanks just fixed that!